tests/wen_new_standard.test.ts

```diff
@@ -0,0 +1,687 @@
+      it("should have no rent", async () => {
+        expect(tokenAccountLamports).to.eql(0);
+      });
+      it("should have no data", async () => {
+        expect(tokenAccountInfo).to.be.null;
+      });
+    });
+  });
+
+  describe("group", () => {
+    const groupAuthorityKeyPair = Keypair.generate();
+    const groupAuthorityPublicKey = groupAuthorityKeyPair.publicKey;
+
+    const groupMintKeyPair = Keypair.generate();
+    const groupMintPublicKey = groupMintKeyPair.publicKey;
+
+    const [group] = PublicKey.findProgramAddressSync(
+      [GROUP_ACCOUNT_SEED, groupMintPublicKey.toBuffer()],
+      program.programId
+    );
+
+    let createGroupArgs;
+    let groupAccountInfo: AccountInfo<Buffer>;
+    let groupAccount;
+
+    before(async () => {
+      createGroupArgs = {
+        name: faker.lorem.text(),
+        symbol: faker.lorem.word(),
+        uri: faker.internet.url(),
+        maxSize: faker.number.int({ min: 1, max: 1_000_000 }),
+      };
+
+      const mintTokenAccount = getAssociatedTokenAddressSync(
+        groupMintPublicKey,
+        groupAuthorityPublicKey,
+        true,
+        TOKEN_2022_PROGRAM_ID
+      );
+
+      await program.methods
+        .createGroupAccount(createGroupArgs)
+        .accountsStrict({
+          mintTokenAccount,
+          mint: groupMintPublicKey,
+          authority: groupAuthorityPublicKey,
+          receiver: groupAuthorityPublicKey,
+          group,
+          associatedTokenProgram: ASSOCIATED_TOKEN_PROGRAM_ID,
+          tokenProgram: TOKEN_2022_PROGRAM_ID,
+          payer,
+          manager,
+          rent: SYSVAR_RENT_PUBKEY,
+          systemProgram: SystemProgram.programId,
+        })
+        .signers([groupMintKeyPair, groupAuthorityKeyPair])
+        .rpc({
+          skipPreflight: true,
+          preflightCommitment: "confirmed",
+          commitment: "confirmed",
+        });
+
+      groupAccountInfo = await program.account.tokenGroup.getAccountInfo(group);
+      groupAccount = program.coder.accounts.decode(
+        "tokenGroup",
+        groupAccountInfo.data
+      );
+    });
+
+    describe("after creating", () => {
+      it("should be an account owned by the program", async () => {
+        expect(groupAccountInfo.owner).to.eql(program.programId);
+      });
+
+      it("should have an update authority", async () => {
+        expect(groupAccount.updateAuthority).to.eql(groupAuthorityPublicKey);
+      });
+
+      it("should reference the group mint", async () => {
+        expect(groupAccount.mint).to.eql(groupMintPublicKey);
+      });
+
+      it("should have a max size", async () => {
+        expect(groupAccount.maxSize).to.eql(createGroupArgs.maxSize);
+      });
+
+      it("should have no members denoted by a size of 0", async () => {
+        expect(groupAccount.size).to.eql(0);
+      });
+    });
+
+    describe("after updating", () => {});
+
+    describe("after adding a mint as a member", () => {
```

Comment:
We should also add a test to ensure minting too many NFTs isn't possible. Might also want to reduce the maximum number in the random generator as well, a million in a for loop minting on localnet could possibly take awhile.

Or could make sense to do a new group of like 5 just to verify explicitly that.